src/main/java/domain/piece/piecerole/Bishop.java

```diff
@@ -0,0 +1,49 @@
+package domain.piece.piecerole;
+
+import static domain.move.Direction.NORTH_EAST;
+import static domain.move.Direction.NORTH_WEST;
+import static domain.move.Direction.SOUTH_EAST;
+import static domain.move.Direction.SOUTH_WEST;
+
+import domain.move.Movable;
+import domain.position.Position;
+import java.util.List;
+import java.util.Objects;
+
+public class Bishop implements PieceRole {
+    private static final int MAX_MOVEMENT = 7;
+
+    private final List<Movable> routes;
+
+    public Bishop() {
+        routes = List.of(
+                new Movable(MAX_MOVEMENT, NORTH_EAST),
+                new Movable(MAX_MOVEMENT, NORTH_WEST),
+                new Movable(MAX_MOVEMENT, SOUTH_EAST),
+                new Movable(MAX_MOVEMENT, SOUTH_WEST)
+        );
+    }
+
+    @Override
+    public boolean canMove(final Position sourcePosition, final Position targetPosition) {
+        return routes.stream()
+                .anyMatch(movable -> movable.canMove(sourcePosition, targetPosition));
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
+        Bishop bishop = (Bishop) o;
+        return Objects.equals(routes, bishop.routes);
```

Comment:
모든 비숍이 같은 `route`를 가질텐데, 그렇다면 다음과 같이 수정해도 괜찮지 않을까 싶네요. 
그런데, 왼쪽에서 시작하는 검정 비숍과 오른쪽에서 시작하는 검정 비숍이 동등하다는 결과가 나오는 것이 의도된 것인가요?
```suggestion
        return true;
```